src/chart/graph/GraphSeries.ts

```diff
@@ -69,7 +69,7 @@ export interface GraphNodeStateOption<TCbParams = never> {
 
 
 interface ExtraEmphasisState {
-    focus?: DefaultEmphasisFocus | 'adjacency'
+    focus?: DefaultEmphasisFocus | 'adjacency' | 'fullPath'
```

Comment:
Do you have other suggestions for the name of this new value? We should avoid using two words because developers may misspell it to be `'fullpath'`.

Reply:
what about `trajectory`? or just `full` ?